docs/networking.md

```diff
@@ -55,8 +55,8 @@ scope driver.
 As you can see here, the ID is the same on the two nodes, because it's the same
 network.
 
-If you want to want to create a local scope network (for example with the bridge
-driver) you should use `<node>/<name>` otherwise your network will be created on a
+If you want to create a local scope network (for example with the bridge
```

Comment:
maybe fix 'global stop driver' up above in the same fashion as this if it is deemed an acceptable change. I'm still not sure if the phrase is odd or not.